api/models/Product/Product.js

```diff
@@ -0,0 +1,22 @@
+const Sequelize = require('sequelize');
+
+const sequelize = require('../../../config/database');
+
+
+const tableName = 'products';
+
+
+const Product = sequelize.define('Product', {
+name: {
+  type: Sequelize.STRING
+},
+description: {
+  type: Sequelize.TEXT,
+},
+price: {
+  type: Sequelize.STRING,
```

Comment:
this should be type INT or FLOAT, check for field types in sequelize for numbers